### PR TITLE
Update performance.md. Mention PPM

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: core/performance.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Performance
+
+## Enabling the metadata cache
+
+Computing metadata used by the bundle is a costly operation. Fortunately, metadata can be computed once and then cached.
+API Platform internally uses a [PSR-6](http://www.php-fig.org/psr/psr-6/) cache. If the Symfony Cache Component is available
+(the default in the standard edition), it automatically enables the support for the best cache adapter available.
+
+Best performance are achieved using [APCu](https://github.com/krakjoe/apcu). Be sure to have the APCu extesnion installed
```
Comment (teohhanhui) [inline core/performance.md]: performance is

File: core/performance.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+on your production server, API Platform will automatically use it.
+
+## Using PPM (PHP-PM)
+
+Response time of the API can be improved up to 15x by using [PHP Process Manager](https://github.com/php-pm/php-pm). If
```
Comment (teohhanhui) [inline core/performance.md]: Is it production ready yet? I think we should leave this out.

Comment (dunglas) [inline core/performance.md]: It is used by large companies in production... IMO it worth mentioning it.

Comment (teohhanhui) [inline core/performance.md]: PPM's `README` seems to suggest otherwise... Hmm... Perhaps a word of caution is still warranted?

Comment (dunglas) [inline core/performance.md]: Warning added.

**Comment by soyuka** [comment]
:+1:

**Pull request merged**